src/gc.c

```diff
@@ -262,8 +262,8 @@ static void check_load(void)
 	if (getloadavg(load, 3) == -1)
 		return;
 
-	// Get number of CPU cores
-	const int nprocs = get_nprocs();
+	// Get number of totally existing CPU cores
```

Comment:
```suggestion
	// Get number of configured CPU cores
```

Just to use the same name as in the location

Reply:
As you see, I'm a bit struggling with the vocabulary here

- Configured means "totally physically existing", but
- Available (sometimes called "online") means something different. It is the number of cores the VM configuration allows to be used

...

Reply:
I see what you mean.  If the function would be called `get_nprocs_exist` it would all be much clearer.
Configured and available are to similar in meaning I think.
___

Maybe we leave "totally existing" in `gc.c` but change L162 in `info.c` to also "totally existing"?

Reply:
Done that and force-pushed over the existing commit